pkg/render/dex_config.go

```diff
@@ -119,27 +121,28 @@ type DexRelyingPartyConfig interface {
 
 func NewDexRelyingPartyConfig(
 	authentication *oprv1.Authentication,
-	tlsSecret *corev1.Secret,
+	caSecret *corev1.Secret,
```

Comment:
Internally I think this is be called certSecret, should this be the same?
Could you add function definitions with some details on the arguments since this function and the NewDexKeyValidatorConfig have had this field changed but NewDexConfig has not changed.